.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+
+- repo: https://github.com/humitos/mirrors-autoflake.git
+  rev: v1.3
+  hooks:
+    - id: autoflake
+      args: ['--in-place', '--expand-star-imports', '--ignore-init-module-imports', '--remove-all-unused-imports']
+
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
```

Comment:
do you want to use up-to-date `20.8b1`? `19.3b0` is from March 2019

Reply:
Appreciate the comments on the pre-commit/tests.sh/buildspec, but these are simply mimicking what is done in our other repos. Making all these changes here would result in inconsistent behavior for the other repos. I will keep track of all these suggestions so that when the time comes for us to update all these files, we will do this for all of our repos at the same time.

Reply:
Going to resolve this and related comments.